recipes/fs-s3fs/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "fs-s3fs" %}
+{% set version = "0.1.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "eae26f226a8db3fe1966b477adeb0369b174093a60dfdb35c63da9c3050a8e49" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - boto3 >=1.4.0,<2
+    - fs >=2.0.7,<3
+    - six >=1.10.0,<2
+    - enum34  # [py2k]
+  run:
+    - python
+    - boto3 >=1.4.0,<2
+    - fs >=2.0.7,<3
+    - six >=1.10.0,<2
+    - enum34  # [py2k]
+
+test:
+  requires:
+    - setuptools
```

Comment:
Why is `setuptools` listed here?

Reply:
Ok, it looks like the project uses `pkg_resources`, so you need `setuptools`, but it should be a `run` requirement, not a specific req for the tests.